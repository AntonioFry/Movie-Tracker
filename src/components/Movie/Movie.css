.movie {
  margin: 10px;
}