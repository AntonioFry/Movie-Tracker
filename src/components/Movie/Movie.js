import React from 'react';
import '../Movie/Movie.css'
import { Link } from 'react-router-dom';

const Movie = ({ poster, id }) => {
  const imgSrc = `http://image.tmdb.org/t/p/w185//${poster}`
  return (
    <Link to={`/${id}`} key={id} className='movie'>
        <img src={imgSrc} alt="movie poster"/>
    </Link>
  )
}

export default Movie;