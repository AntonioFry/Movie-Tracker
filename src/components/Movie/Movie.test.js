import React from 'react';
import { shallow } from 'enzyme';
import Movie from './Movie';

describe('Movie', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Movie poster='' id={1} />
    )
  })
  it('should match snapshot', () => { 
    expect(wrapper).toMatchSnapshot()
  }); 
});


