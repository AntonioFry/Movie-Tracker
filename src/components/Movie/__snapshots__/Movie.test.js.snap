// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie should match snapshot 1`] = `
<Link
  className="movie"
  key="1"
  to="/1"
>
  <img
    alt="movie poster"
    src="http://image.tmdb.org/t/p/w185//"
  />
</Link>
`;
exports[`Movie should match the snapshot 1`] = `ShallowWrapper {}`;
